mobile/lib/pages/article_selection_page.dart

```diff
@@ -48,11 +49,13 @@ class ArticleSelectionPage extends HookWidget {
       ), // Helps with the player not covering the last article
       child: GridView.builder(
         controller: hooks.scrollController,
-        padding: const EdgeInsets.all(20),
+        padding: RaPageConstraints.outerWidgetPagePadding.copyWith(
+          top: RaPageConstraints.pagePadding,
+        ),
```

Comment:
One last thing: add a bottom padding here (possibly `RaPageConstraints.radioPlayerPadding`), because it clashes with the radio player when scrolled to the bottom